packages/postcss-tailwind-css-modules/tsdown.config.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { defineConfig } from 'tsdown';
+
+export default defineConfig({
+  plugins: [],
+  entry: [
+    'src/**/*.{ts,tsx}',
+    '!src/**/*.{d,stories,test,test-d,bench}.{ts,tsx}',
+    '!**/__fixtures__',
+  ],
+  clean: true,
+  dts: true,
+  format: ['esm', 'cjs'],
+  sourcemap: true,
+  unbundle: true,
+  treeshake: true,
+  platform: 'neutral',
+  minify: false,
+  exports: {
+    customExports(pkg) {
+      pkg['.'] = {
+        // ensure the order matches syncpack for consistent ordering
+        import: pkg['.'].import,
+        require: pkg['.'].require,
+      };
+      return pkg;
+    },
```

Comment:
tsdown had  import  and require in the wrong order. it conflicted w/ syncpack, causing formatting inconsistencies, but it also does not follow the spec.

https://publint.dev/rules#imports_module_should_precede_require
https://nodejs.org/api/packages.html#conditional-exports

i plan to open a tsdown issue for this, but the workaround for now is to fix the order manually